src/Materials/Textures/cubeTexture.ts

```diff
@@ -389,6 +304,48 @@ export class CubeTexture extends BaseTexture {
         this._textureMatrix = value;
     }
 
+    private loadTexture(onLoad: Nullable<() => void> = null, onError: Nullable<(message?: string, exception?: any) => void> = null) {
+        const scene = this.getScene();
+        const oldTexture = this._texture;
+        this._texture = this._getFromCache(this.url, this._noMipmap, undefined, undefined, this._useSRGBBuffer);
+
+        let onLoadProcessing = () => {
+            this.onLoadObservable.notifyObservers(this);
+            if (oldTexture) {
+                oldTexture.dispose();

```

Comment:
I think this call is missing (that was done in `updateURL` previously):
```typescript
this.getScene()?.markAllMaterialsAsDirty(Constants.MATERIAL_TextureDirtyFlag);
```

Reply:
Thanks, added :-)